src/core/trulens/core/session.py

```diff
@@ -193,7 +188,6 @@ def __init__(
                 logger.warning(
                     "TruSession was already initialized. Cannot change database configuration after initialization."
                 )
-                self.warning()
             return
```

Comment:
Is the whole condition not required any more? It was for the singletonbyname to abort a second call to `__init__` but if I understand right, the metaclass will prevent the second init now?

Reply:
Condition starting `if python.safe_hasattr(self, "connector"):`.

Reply:
Although it would be nice to have a warning in case someone wants to reconfigure the session. See another comment about the warnings.

Reply:
Yea this would only invoke `__init__` once. We can throw a warning though